tools/gmon/gmon/clients/utils.py

```diff
@@ -0,0 +1,47 @@
+"""
+`utils.py`
+Util functions.
+"""
+
+from google.protobuf.json_format import MessageToDict
+
+
+def decorate(decorator, methods=[]):
```

Comment:
I'd personally do something like this

```python3

class DecorateMethods:
    def __init__(self, methods: Set[str]):
        self.methods = methods

    def with(decorator: Callable):
          def inner(cls): 
              for attr in cls.__dict__:
                  attribute = getattr(cls, attr)
              if callable(attribute) and attribute.__name__ in methods:
                  setattr(cls, attr, decorator(attribute))
              return cls

          return inner
```
This way usage reads like english
```python3
@DecorateMethods({'foo', 'bar'}).with(to_json)
```

Reply:
This looks great, but I'm getting into an error when doing this: 
```
@decorate_methods({'get', 'create', 'list', 'delete', 'inspect'}).with(to_json)
                                                                                                      ^
SyntaxError: invalid syntax
```

Reply:
Also tried to rename `with` (Python keyword) to `on`, but doing this fails to decorate the functions:
```
decorator = decorate_methods({'get', 'create', 'list', 'delete',
                              'inspect'}).on(to_json)


@decorator
class MetricsClient:
```

Reply:
How about renaming my existing decorator to `decorate_with`, so it can be used like:
```
@decorate_with(to_json, methods=['get', 'create', 'list', 'delete', 'inspect'])
class MetricsClient
```

Reply:
this makes a lot of sense and is in fact simpler.